config/sidekiq.yml

```diff
@@ -19,3 +19,7 @@ production:
     every: 12h
   UploadRemovalWorker:
     every: 12h
+  TheTvdbDailyWorker:
```

Comment:
@NuckChorris I am not sure if this is valid actually.

Reply:
I just confirmed that it's fine